src/actions/component-remove.js

```diff
@@ -25,9 +26,9 @@ export function removeComponentFromActivesComponent(
     if (currentId !== deletedComponent.id) {
       acc[currentId] = {
         ...activesComponents[currentId],
-        children: activesComponents[currentId].children.filter(
+        children: activesComponents[currentId] && activesComponents[currentId].children? activesComponents[currentId].children.filter(
```

Comment:
Missing space. 
 
In a new version of Babel, we can now use the elvis operator ?. . 

This operator is usefull if we want to avoid this kind of syntax. 

We should maybe update babel dependencies if doable.